sonic-xcvrd/xcvrd/xcvrd_utilities/media_settings_parser.py

```diff
@@ -89,6 +111,10 @@ def get_media_settings_key(physical_port, transceiver_dict, port_speed, lane_cou
             media_compliance_dict = ast.literal_eval(media_compliance_dict_str)
             if sup_compliance_str in media_compliance_dict:
                 media_compliance_code = media_compliance_dict[sup_compliance_str]
+                # For 100G transceivers, it's usually in extended specification compliance
+                if media_compliance_code in sup_compliance_extended_values and \
```

Comment:
You are overwriting media_compliance_code. Can we update only if its has "Extended" or "Unknown" ?

Reply:
yes, updated